src/util/mod.rs

```diff
@@ -35,8 +35,8 @@ pub fn make_spirv<'a>(data: &'a [u8]) -> super::ShaderModuleSource<'a> {
     // otherwise copy the byte array in an owned vector and use that instead.
     let words = if data.as_ptr().align_offset(align_of::<u32>()) == 0 {
         let (pre, words, post) = unsafe { data.align_to::<u32>() };
-        debug_assert_eq!(pre, &[]);
-        debug_assert_eq!(post, &[]);
+        debug_assert!(pre.is_empty());
```

Comment:
why this change?

Reply:
Remember that issue I had like 2 weeks ago or so, where if you build with the subscriber feature this line of code fails to typecheck? That apparently made it into master, so this fixes this. We need to add the subscriber feature to ci.